.github/ISSUE_TEMPLATE/vetsapi-argo-terminal-access.yaml

```diff
@@ -0,0 +1,132 @@
+name: Vets-api ArgoCD terminal access
+description: To request access to vets-api terminal
+title: Vets-api terminal access for [individual]
+labels: ['external-request', 'platform-tech-team-support', 'ops-access-request']
```

Comment:
Are these labels okay? I want to make sure we're getting this right because the labels are passed in through the URL, so if we have to change the labels, we have to also update any place we have this issue template linked. (not a huge deal, but annoying!)

Reply:
LGTM